testdata/jsight_0.3/req.japi.enum/valid.json

```diff
@@ -0,0 +1,76 @@
+{
+  "tags": {},
+  "userTypes": {
+    "@foo": {
+      "schema": {
+        "content": {
+          "tokenType": "object",
+          "type": "object",
+          "children": [
+            {
+              "key": "foo",
+              "tokenType": "number",
+              "type": "enum",
+              "scalarValue": "42",
+              "rules": [
+                {
+                  "key": "enum",
+                  "tokenType": "reference",
+                  "scalarValue": "@enum"
+                }
+              ],
+              "optional": false
+            }
+          ],
+          "optional": false
+        },
+        "notation": "jsight"
+      }
+    }
+  },
+  "userEnums": {
+    "@enum": {
+      "annotation": "",
+      "description": "",
+      "value": {
+        "key": "@enum",
+        "tokenType": "object",
+        "children": [
+          {
+            "key": "\"foo\"",
```

Comment:
I think there shouldn't be double quotes.

Reply:
Done